src/extension/background-script/index.ts

```diff
@@ -141,6 +141,24 @@ browser.runtime.onInstalled.addListener(handleInstalled);
 async function init() {
   console.info("Loading background script");
 
+  const registerInPageContentScript = async () => {
+    try {
+      await chrome.scripting.registerContentScripts([
+        {
+          id: "inpage",
+          matches: ["file://*/*", "http://*/*", "https://*/*"],
+          js: ["js/inpageScript.bundle.js"],
+          runAt: "document_start",
+          world: "MAIN",
+        },
+      ]);
+    } catch (err) {
+      console.warn(`Dropped attempt to register inpage content script. ${err}`);
```

Comment:
Does the app still work if this fails?

Reply:
for chromium based browser? i don't think this fails. it's just that we are registering script programatically instead of using manifest. but core thing is same.

for other browsers. if function fails rest of the flow will still continue. on firefox
```
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/c4a562d7-a94e-4a47-af62-84d4ae27ae18)

```

Reply:
resolved